### PR TITLE
2fly2

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

**Comment by akallabeth** [comment]
reformatted #9285

**Comment by pnowack** [comment]
I haven't checked the client side code here, but we should check the thinclient + small cache option first. If both are set, then the small cache flag should not be set in the first place (that is mentioned somewhere in `[MS-RDPEGFX]`.

**Comment by akallabeth** [comment]
@pnowack reformated the commit message. as for the flag, I´ve checked and it should not be the default setting as that affects both, client and server. your suggestion should already be implemented, but I need to verify that.

File: libfreerdp/core/settings.c [hunk 1]
```diff
@@ -648,7 +648,7 @@ rdpSettings* freerdp_settings_new(DWORD flags)
 	    !freerdp_settings_set_bool(settings, FreeRDP_NSCodecAllowDynamicColorFidelity, TRUE) ||
 	    !freerdp_settings_set_bool(settings, FreeRDP_AutoReconnectionEnabled, FALSE) ||
 	    !freerdp_settings_set_uint32(settings, FreeRDP_AutoReconnectMaxRetries, 20) ||
-	    !freerdp_settings_set_bool(settings, FreeRDP_GfxThinClient, TRUE) ||
+	    !freerdp_settings_set_bool(settings, FreeRDP_GfxThinClient, FALSE) ||
```
Comment (hardening) [inline libfreerdp/core/settings.c:651]: Shall we document that in the merging to FreeRDP 3.0 doc, as in some cases that can be a pretty important detail (those who have played with egfx proxying know what I'm talking about)

Comment (akallabeth) [reply libfreerdp/core/settings.c:651]: @hardening which document would that be? for the client command line that would now fit the doc as that was wrong.

Comment (hardening) [reply libfreerdp/core/settings.c:651]: Oh correct forget my remark

**Pull request merged**